pkg/testing/matrix/matrix.go

```diff
@@ -226,7 +226,12 @@ func Test(t *testing.T, opts TestOptions) {
 		assert.NoError(t, err)
 		assert.NotNil(t, provider)
 
-		schemaBytes, err := provider.GetSchema(int(plugin.Version.Major))
+		max := plugin.Version.Major
+		if max > 1 {
+			max = 1
+		}
+
+		schemaBytes, err := provider.GetSchema(int(max))
```

Comment:
Isn't this just always 1?

Reply:
I think it can be zero? Otherwise why would GetSchema take any params at all?

Reply:
@AaronFriel or @iwahbe might know this for sure

Reply:
As far as I'm aware, it is always 0. This is how we generally call `GetSchema`:
https://github.com/pulumi/pulumi/blob/f618cf8c61a4df51e5e9a2741b3b5d05bbdd21d8/pkg/codegen/schema/loader.go#L246-L247

I assume that the parameter is just to allow us to future proof against breaking schema changes.